.azure-pipelines/steps/run.yml

```diff
@@ -101,6 +102,7 @@ steps:
 
 - bash: |
     set -e
+    rustup self uninstall -y || true
```

Comment:
Can you add the reason why rustup is being uninstalled in a comment? r=me otherwise!